Fix #682 - Store blobs as bytea in PostgreSQL

lehmanju opened the pull request:
This add fixes blobs in PostgreSQL which need to be stored as bytes.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/requery/requery?pullRequest=683) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/requery/requery?pullRequest=683) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/requery/requery?pullRequest=683) it.</sub>

lehmanju commented:
I am sorry, it took some time, had some issues with my internet connection.

I changed imports as requested and added a "write" function that parses blobs to binary streams, otherwise it would not have worked.

npurushe commented on build.gradle:
```diff
@@ -76,14 +76,14 @@ configure([project(':requery'), project(':requery-processor'), project(':requery
-                credentials {
-                    username = bintray.user
-                    password = bintray.key
+                url "https://maven.devpi.de/repository/default"
```
Probably want to revert this file

lehmanju commented:
Yes, thank you ;) I have already changed my password.

npurushe commented:
Thanks!
